app/src/test/java/com/example/campusguide/utils/CalculatorTest.kt

```diff
@@ -0,0 +1,16 @@
+package com.example.campusguide.utils
+
+import org.junit.Test
+
+import org.junit.Assert.*
+
+class CalculatorTest {
+
+    @Test
```

Comment:
I never know when I should use hardcoded variable and when I should use test case data

Reply:
Hardcoded variable is fine here, makes it clear what the inputs and expected outputs are